BrigetteDoelp/fitlit-starter-kit
# Hydration stats on dashboard & some small tasks

## What’s this PR do?
​* Call `HydrationRepo` methods in scripts.js to display hydration data on DOM
* Rename files to reflect class naming
* Rename variables in HydrationRepo to be more concise
* Automate PR template to show up after this PR
## Where should the reviewer start?
​* scripts.js
## How should this be manually tested?
​run `open src/index.html` in your terminal to get page up in your browser; last couple lines should awkwardly display hydration stats for user 33.
## What are the relevant tickets?
​Resolves #12 
Resolves #13 
Resolves #28 
Resolves #31

## Questions:
n/a
